styles.css

```diff
@@ -1,50 +1,126 @@
 
 
-
-
 body {
   background-color: #c1cdcd;
   margin: 0;
   padding: 0;
-  font-family: Georgia, 'Times New Roman', Times, serif;
   display: grid;
-  grid-template-columns: 1fr 3fr;
+  grid-template-columns: 1fr 4fr;
   grid-template-areas: "sidebar content";
+  font-family: Caveat, Georgia, 'Times New Roman', Times, serif;
+  font-size: 1.1em;
 }
 
 nav {
-  background-color: #c1ffc1;
+  background-color: #ffefdb;
   grid-area: sidebar;
   display: grid;
   grid-template-columns: 1fr;
   grid-template-rows: 40px auto;
   min-height: 100vh;
+  min-width: 270px;
+  font-family: "Grenze Gotisch", Times, serif;
 }
 
 nav>header {
-  background-color: #7ccd7c;
+  background-color: #8b2323;
+  padding: 10px 10px 20px 10px;
+  min-height: 1em;
+  font-size: 1.6em;
+  font-weight: 600;
+  color: #ffec8b;
+}
+
+nav>ul {
+  margin-top: 2.5em;
 }
 
 nav>footer {
-  background-color: #adcd7c;
-  height: 60px;
-  font-size: 80%;
+  background-color: #ab4343;
+  height: 4em;
+  padding: 0px 10px 10px 10px;
+  font-size: 0.8em;
+  color: #fff;
   align-self: end;
 }
 
 main {
-  background-color: #fdf5e6;
+  background-color: #8b7d6b;
   grid-area: content;
+  padding: 0 15px;
 }
 
 section {
-  background-color: #eec591;
+  background-color: #ffefdb;
+  margin: 5px 0 20px 0;
+  padding: 1em;
+  border-radius: 5px;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.6);
+  color: #4b3305;
 }
 
 section>header {
   background-color: #8b3e2f;
+  padding: 5px 10px;
+  margin: -0.6em -0.6em 0.6em -0.6em;
+  border-radius: 2px;
+  font-family: "Grenze Gotisch", Times, serif;
+  font-size: 1.4em;
+  font-weight: 600;
+  color: #fdd891;
+}
+
+article p:first-child::first-letter {
+  color: 8b2323;
+  padding: 0 .3rem;
+  margin: .5rem .2rem 0 -0.2rem;
+  font-family: "Grenze Gotisch", Times, serif;
+  font-size: 3rem;
+  float: left;
+  line-height: 1;
+}
+
+strong {
+  color: #8b2323;
+}
+
+a {
+  color: #8b2323;
+}
+
+a:hover {
+  animation-name: link-color;
+  animation-duration: 700ms;
+  animation-fill-mode: forwards;
 }
 
+@keyframes link-color {
```

Comment:
I am not sure I understand what this code is doing. Is this just changing the colour when the mouse button hovers? I see the duration is 700ms but I can't really appreciate a fade-in compared with just having this (it might be I'm sleepy)
```
a:hover {
  color: #ab4848;
}
```
Just wondering about this

Reply:
Changing the base color to a darker one and adding some movement so the anim. is more noticeable. It is just a bit of a test to try out animations.